samples/apps/cap/py/autogencap/ag_adapter/AG2CAP.py

```diff
@@ -4,7 +4,7 @@
 from autogen import Agent, ConversableAgent
 
 from ..actor_runtime import IRuntime
-from .AutoGenConnector import AutoGenConnector
+from .autogen_connector import AutoGenConnector
 
 
 class AG2CAP(ConversableAgent):
```

Comment:
Would you like to also rename this file to something like "ag2cap.py"?

Reply:
Yes.  Makes sense.  Thanks.

Reply:
I am reconsidering some abstractions for the actor itself.  Converting this to draft.  Thanks @thinkall